src/datasets/commands/dummy_data.py

```diff
@@ -8,6 +8,8 @@
 from pathlib import Path
 from typing import Optional
 
+import ijson
```

Comment:
ijson is an optional dependency so this import would crash for the users that don't have ijson

Can you move the import inside the `_create_json_dummy_data` method and raise an error if it's not installed ?
The error should also tell the user to install ijson with `pip install ijson` to be able to create dummy json files.

Reply:
@lhoestq done. I have also fixed the conflicts it had with the master branch.